src/astro/sql/operators/append.py

```diff
@@ -4,12 +4,13 @@
 from airflow.models.baseoperator import BaseOperator
 
 from astro.databases import create_database
+from astro.sql.operators.upstream_tasks import UpstreamTaskMixin
 from astro.sql.table import Table
 
 APPEND_COLUMN_TYPE = Optional[Union[List[str], Tuple[str], Dict[str, str]]]
 
 
-class AppendOperator(BaseOperator):
+class AppendOperator(UpstreamTaskMixin, BaseOperator):
```

Comment:
Does this PR make all of the Python Astro SDK decorators and tasks inherit from `UpstreamTaskMixin`?

Would it make more sense to define an Astro SDK `BaseOperator` class which would both:
* inherit from `BaseOperator`
* implement the `UpstreamTaskMixing` behaviour that should be available to all operators?

Reply:
@tatiana I've made an AstroSQLBaseOperator class that takes in the UpstreamTaskMixin. This will look cleaner for those implementing operators, but we have to keep the mixin because the `BaseSQLDecoratedOperator` also depends on the `DecoratedOperator` class so we would run the risk of double inheritance.